spec/intl.html

```diff
@@ -93,32 +148,46 @@ <h1>
+            The List is sorted by [[Identifier]] of each Time Zone Identifier Record.
+          </dd>
           <dt>redefinition</dt>
           <dd>true</dd>
         </dl>
         <emu-alg>
-          1. Let _timeZones_ be a List containing the String value of each Zone or Link name in the IANA Time Zone Database that is supported by the implementation.
-          1. Assert: _timeZones_ contains *"UTC"*.
-          1. Assert: _timeZones_ does not contain any element that does not identify a Zone or Link name in the IANA Time Zone Database.
-          1. Return SortStringListByCodeUnit(_timeZones_).
+          1. Let _identifiers_ be a List containing the String value of each Zone or Link name in the IANA Time Zone Database that is supported by the implementation.
+          1. Assert: No element of _identifiers_ is an ASCII-case-insensitive match for any other element.
+          1. Assert: No element of _identifiers_ does not identify a Zone or Link name in the IANA Time Zone Database.
+          1. Set _identifiers_ to SortStringListByCodeUnit(_identifiers_).
+          1. Let _result_ be a new empty List.
+          1. For each element _identifier_ of _identifiers_, do
+            1. Let _primary_ be _identifier_.
+            1. If _identifier_ is a non-primary time zone identifier and _identifier_ is not *"UTC"*, then
+              1. Set _primary_ to the name of the primary time zone identifier that _identifier_ resolves to, according to the rules for resolving Link names in the IANA Time Zone Database.
+              1. NOTE: If _identifier_ resolves to another non-primary time zone identifier, then the implementation must continue resolution of the entire chain until its terminal primary time zone identifier.
+            1. If _primary_ is one of *"Etc/UTC"*, *"Etc/GMT"*, or *"GMT"*, set _primary_ to *"UTC"*.
+            1. Let _record_ be the Time Zone Identifier Record { [[Identifier]]: _identifier_, [[PrimaryIdentifier]]: _primary_ }.
+            1. Append _record_ to the end of _result_.
+          1. Assert: One element of _result_ is the Time Zone Identifier Record { [[Identifier]]: *"UTC"*, [[PrimaryIdentifier]]: *"UTC"* }.
+          1. Return _result_.
         </emu-alg>
 
-        <p>This definition supersedes the definition provided in <emu-xref href="#sec-availabletimezones"></emu-xref>.</p>
+        <emu-note>
+          Time zone identifiers in the IANA Time Zone Database can change over time.
+          At a minimum, it is recommended that implementations limit changes to the result of AvailableTimeZoneIdentifiers to the changes allowed by GetAvailableTimeZoneIdentifier, for the lifetime of the surrounding agent.
+          Due to the complexity of supporting these requirements, it is recommended that the result of AvailableTimeZoneIdentifiers remains the same for the lifetime of the surrounding agent.
```

Comment:
Same nitpick here, it's not a requirement, but should it be?

Reply:
Sorry for misunderstanding the first time. Fixed now.